src/sidebar/services/tags.js

```diff
@@ -21,7 +21,7 @@ export default function tags(localStorage) {
    * Return a list of tag suggestions matching `query`.
    *
    * @param {string} query
-   * @param {number} limit - Optional limit of the results.
+   * @param {number|null} limit - Optional limit of the results.
```

Comment:
`@param {number|null} [limit=null]`

Reply:
I didn't specify a default here because TS infers it automatically from the `= null` in the parameter list below.